storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -530,6 +559,10 @@ private static boolean downloadResourcesAsSupervisorAttempt(ClientBlobStore cb,
         return isSuccess;
     }
 
+    public static boolean checkFileExists(String path) {
+        return Files.exists(new File(path).toPath());
+    }
```

Comment:
Can we file a JIRA to split the file system utils out of Utils.  There are a lot of these and they should be in their own java file.